tools.mk

```diff
@@ -65,6 +71,27 @@ $(PROTOBUF):
 	@echo "Get GoGo Protobuf"
 	@go get github.com/gogo/protobuf/protoc-gen-gogo@v1.3.1
 
+buf: protoc-gen-buf-check-breaking protoc-gen-buf-check-lint
+	@echo "Installing buf..."
+	@curl -sSL \
+    "https://github.com/bufbuild/buf/releases/download/v${BUF_VERSION}/buf-${UNAME_S}-${UNAME_M}" \
+    -o "${BIN}/buf" && \
+	chmod +x "${BIN}/buf"
+
+protoc-gen-buf-check-breaking:
```

Comment:
FYI @marbar3778 there's no need for you to install `protoc-gen-buf-check-breaking` or `protoc-gen-buf-check-lint` for your usage of `buf check lint` and `buf check breaking` above - these are for if you want to do i.e. `protoc --buf-check-breaking_out=.`, which is primarily for use cases such as Bazel.